windows/hid.c

```diff
@@ -351,14 +349,9 @@ struct hid_device_info HID_API_EXPORT * HID_API_CALL hid_enumerate(unsigned shor
 			if (!res)
 				goto cont;
 
-			if (strcmp(driver_name, "HIDClass") == 0) {
```

Comment:
Alright, I put it back in. I'm assuming it's meant to try the next device if no driver is found, is this the correct logic?